templates/frontend/react/frontend/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "license": "(MIT AND Apache-2.0)",
   "scripts": {
-    "start": "echo The app is starting! && env-cmd -f ../contract/neardev/dev-account.env parcel index.html --open",
+    "start": "echo The app is starting! && env-cmd --silent -f ../contract/neardev/dev-account.env parcel index.html --open",
```

Comment:
Here is where I would check if the file exists and tell them if it doesn't since the user is still in the console.

Reply:
It'll make the command very complicated, and it's anyway a good idea to keep the check in client-code for the user to see it there, so that it'll be clear that he needs must have a correct contract name.